cpp/arcticdb/processing/expression_node.hpp

```diff
@@ -42,19 +43,23 @@ class StringPool;
 struct ColumnWithStrings {
     std::shared_ptr<Column> column_;
     const std::shared_ptr<StringPool> string_pool_;
+    ColumnName column_name_;
```

Comment:
We may as well store this as a `std::string`, we only need `ColumnName` when it is ambiguous whether the string is a column name, value, expression, etc

Reply:
Fixed in [a907db9](https://github.com/man-group/ArcticDB/pull/1557/commits/a907db94c4366371434d048c630f796fcbc7a60e)